.github/workflows/renovate-self-hosted.yaml

```
@@ -0,0 +1,135 @@
+
+permissions: {}  # least-privilege; Renovate uses RENOVATE_TOKEN
+
+on:  # yamllint disable-line rule:truthy
+  workflow_dispatch:
+    inputs:
+      renovate_dry_run:
+        description: >-
+          Leave empty for a normal run (branches and PRs). Otherwise pick a Renovate dry-run mode.
+        type: choice
+        options:
+          - ""
```

actionlint (1.7.12)

[error] string should not be empty

(syntax-check)